packages/runtime/src/controls/style-v2.ts

```diff
@@ -0,0 +1,79 @@
+import { BoxModel } from 'css-box-model'
+import { ControlDefinition, ControlDefinitionData } from './control'
+import { PropController } from '../prop-controllers/base'
+import { Send } from '../prop-controllers/instances'
+import { ResponsiveValue } from './types'
+import { CSSObject } from '@emotion/serialize'
+import { useStyle } from '../runtimes/react/use-style'
+
+export const StyleV2ControlType = 'makeswift::controls::style-v2'
+
+export const StyleV2Format = {
+  Class: 'makeswift::controls::style-v2::format::class',
+  Object: 'makeswift::controls::style-v2::format::object',
+} as const
+
+export type StyleV2Format = typeof StyleV2Format[keyof typeof StyleV2Format]
+
+export type StyleV2CSSObject = CSSObject
+export const useUnstableStyleV2ClassName = useStyle
```

Comment:
Can we keep "unstable" as a prefix always: i.e., `unstable_useStyleV2ClassName` so that we can easily find it across the codebase?